docs/reference/buildx_bake.md

```diff
@@ -25,6 +25,7 @@ Build from a file
 | [`--pull`](#pull) |  |  | Always attempt to pull all referenced images |
 | `--push` |  |  | Shorthand for `--set=*.output=type=registry` |
 | [`--set`](#set) | `stringArray` |  | Override target value (e.g., `targetpattern.key=value`) |
+| `--sync-output` |  |  | Ensure all builds complete before beginning output |
```

Comment:
Maybe `--sync` would be enough? There are some cases where we might not need to output anything but keep build result synced (`type=cacheonly`)?

```suggestion
| `--sync` |  |  | Ensure all builds complete before returning result |
```

Let's also add a simple example here for this flag.

Reply:
I think `--sync` isn't enough, you want to hint it is about synching the 'stuff at the end'